nvflare/fuel/utils/pipe/cell_pipe.py

```diff
@@ -178,18 +179,22 @@ def __init__(
         super().__init__(mode)
         self.logger = logging.getLogger(self.__class__.__name__)
 
+        self.site_name = site_name
+        self.token = token
+        self.root_url = root_url
+        self.secure_mode = secure_mode
+        self.workspace_dir = workspace_dir
+
+        sysvarname_placeholders = ["{" + varname + "}" for varname in dir(SystemVarName)]
+        if any([arg in sysvarname_placeholders for arg in [site_name, token, root_url, secure_mode, workspace_dir]]):
+            return
```

Comment:
what is this checking for?

Are we assuming if we see "{SITE_NAME}" or any system var then we are in JOB API mode so don't build the cell?
This seems a bit hacky to me though

Reply:
Yes this is needed to prevent the Cell from being built in the JobAPI with the SystemVarNames as arguments.

Reply:
got it, we can think of enhancements later.